graphql/mappers.js

```diff
@@ -330,7 +330,7 @@ const mapParliamentarian = exports.mapParliamentarian = (raw, t) => {
       }
     } : null,
     canton: raw.kanton_name,
-    active: !councilExitDate,
+    active: raw.aktiv,
```

Comment:
The API seems to return `"1"` or `"0"` as a string, so we'll need typecasting (String -> Number -> Bool) or strict equality check.

Typecasting: 
```suggestion
    active: !!+raw.aktiv,
```

This is pretty stable because:
```js
!!+"0" === false
!!+0 === false
!!+false === false
```

What happens:
```js
+"0" === 0
!!0 === false
```

Strict equality check:
```suggestion
    active: raw.aktiv === '1',
```
This would break if the API would start to return a boolean or number down the road.